usagov/vote-gov-nvrf-app
# VOTE-2426 Add Spanish PDF

https://cm-jira.usa.gov/browse/VOTE-2426

Description:
1. Added the new PDF for Spanish (downloaded from eac.gov on 8/29/24)
2. Added a temporary duplicate function for Generating the Spanish PDF. This was done to account for PDF field type differences for Spanish. This will be reverted in the future once the PDF is reconciled with the English PDF.

Test steps:
1. Load the app in your local environment
2. Fill out all fields and confirm the PDF generates for English with all data mapped to the fields.
3. In Delivery.jsx, line 11, hardcode the lang attribute as "es".
4. Fill out all fields and confirm the PDF generates for Spanish with all data mapped to the fields. Pay particular attention to the eligibility fields, title fields, and suffix fields.
NOTE: the app content will still be in English. This will be tested in a separate PR.
